DD4.18 dev velocity content (#16853)
docker/docs: desktop/use-desktop/container.md

- Select **Inspect** to view low-level information about the container. You can see the local path, version number of the image, SHA-256, port mapping, and other details.

> - Select **Files** to explore all the files in your running or stopped containers. You can also:

Reviewer: ```suggestion
- Select **Files** to explore the filesystem of running or stopped containers. You can also:
```
